src/main/java/com/cburch/logisim/std/memory/TFlipFlop.java

```diff
@@ -73,7 +73,7 @@ public String ComponentName() {
     @Override
     public ArrayList<String> GetUpdateLogic() {
       return (new LineBuffer())
-          .add("%s s_next_state %s s_current_state_reg %sT;", HDL.assignPreamble(), HDL.assignOperator(), HDL.xorOperator())
+          .add("{{1}} s_next_state {{2}} s_current_state_reg {{3}}T;", HDL.assignPreamble(), HDL.assignOperator(), HDL.xorOperator())
```

Comment:
Here a space is missing between {{3}} and T

Reply:
Fixed